concrete/controllers/panel/page/check_in.php

```diff
@@ -81,6 +81,10 @@ public function submit()
         if ($this->validateAction()) {
             $comments = $this->request->request('comments');
             $comments = is_string($comments) ? trim($comments) : '';
+            if (strlen($comments) > 255) {
```

Comment:
What about using `mb_strlen()` instead of `strlen()`? They are very different - see https://3v4l.org/5SuGO and https://extendsclass.com/mysql/3325b38